src/toil_scripts/__init__.py

```diff
@@ -19,3 +21,16 @@ def download_from_s3_url(file_path, url):
         key.get_contents_to_filename(file_path)
     finally:
         s3.close()
+
+
+def generate_file(file_path, generate_func):
```

Comment:
Ya, I'd prefer to fix it in this PR. I'll go ahead and PR 29 against the release/1.0.x branch. Once that's merged and a new release is on PyPi I'll make the change on this PR.